### PR TITLE
Suppress 3rd party tags for ad-free subs

### PR DESCRIPTION
**Pull request by JustinPinner**
## What does this change?
Ad-free (digi-pack) subscribers should not have third party tags/cookies dropped. While changing the client-side code for this we can also take advantage of the new no-ads flavour of YouTube videos, so that's included here too. Actually, without this it'd be impossible to continue our ad-free workaround for videos because we were dependent on DFP to make that happen. 

## Screenshots
N/A

## What is the value of this and can you measure success?
We don't waste time loading and running modules pointlessly and we don't drop things on reader's machines.

### Does this affect other platforms?

- [ ] AMP 
- [ ] Apps
- [ ] Other (please specify)
- [x] None of the above

### Does this affect GLabs Paid Content Pages? Should it have support for Paid Content?

- [x] No
- [ ] Yes (please give details)

### Does this change break ad-free?

- [x] No - it makes it BETTER
- [ ] It did, but tests caught it and I fixed it
- [ ] It did, but there was no test coverage so I added that then fixed it

### Does this change update the version of CAPI we're using?

- [x] No, all the existing database files are just fine
- [ ] Yes, and I have [re-run all the tests locally and checked in all the updated data/database/xyz files](https://github.com/guardian/frontend/blob/master/docs/03-dev-howtos/15-updating-test-database.md)

### Accessibility test checklist

- [ ] [Tested with screen reader](https://accessibility.gutools.co.uk/testing/web/screen-readers/)
- [ ] [Navigable with keyboard](https://accessibility.gutools.co.uk/testing/web/keyboard-navigation/)
- [ ] [Colour contrast passed](https://accessibility.gutools.co.uk/testing/web/colour-contrast/)
- [x] There are no accessibility changes to test

### Tested

- [ ] Locally
- [x] On CODE (optional)

### CODE REVIEW

**Comment by janua** [comment]
This is great!  We have talked about doing exactly this (and have already tested it funnily enough) but as it was out of scope we didn't continue doing it. So glad to see this happen. 👏 

Are we sure we don't depend on Admanager/DFP for any reader revenue things such as epics?

Good shout on the listeners too, it's so easy to forget about them.

File: static/src/javascripts/projects/commercial/modules/third-party-tags/doubleclick-ad-free.js [hunk 1]
```diff
@@ -1,20 +0,0 @@
...
-
-const doubleClickRandom = (): string => {
-    const axel = Math.random();
-    const a = axel * 10000000000000;
-    return a.toString();
-};
-
-export const doubleClickAdFree: ThirdPartyTag = {
-    shouldRun:
-        commercialFeatures.adFree &&
```
Comment (johnduffell) [inline static/src/javascripts/projects/commercial/modules/third-party-tags/doubleclick-ad-free.js]: so we used to put ad free users in a special bucket for ad free identified by the random number, but now we don't show ads for anyone so this isnt' needed any more?

Comment (johnduffell) [reply static/src/javascripts/projects/commercial/modules/third-party-tags/doubleclick-ad-free.js]: I mean for youtube specifically as this seems to reference a youtube switch

**Comment by johnduffell** [comment]
great, looks good to my untrained eye, some nice reformatting to keep things a bit tidier too! 👍

File: static/src/javascripts/projects/commercial/modules/third-party-tags/doubleclick-ad-free.js (hunk 1 above)
Comment (JustinPinner) [reply static/src/javascripts/projects/commercial/modules/third-party-tags/doubleclick-ad-free.js]: Yep, exactly that!

**Pull request merged**